maps/sccv_horizon/sccv_horizon-1_deck_1.dmm

```diff
@@ -17629,6 +17629,7 @@
 /area/maintenance/operations)
 "olu" = (
 /obj/structure/flora/ausbushes/reedbush,
+/obj/item/holder/chicken,
```

Comment:
You added the chicken *item*, not a chicken mob.

Reply:
What in the goddamn